lib/core/src/lib/form/components/widgets/date-time/date-time.widget.spec.ts

```diff
@@ -79,14 +82,17 @@ describe('DateTimeWidgetComponent', () => {
     });
 
     it('should setup max value for date picker', () => {
-        const maxValue = '1982-03-13T10:00:000Z';
+        let maxValue = '1982-03-13T10:00:000Z';
```

Comment:
should be const as you are not changing it

Reply:
changed back to const